pyproject.toml

```diff
@@ -6,8 +6,9 @@ build-backend = "hatchling.build"
 name = "qiskit-addon-sqd"
 version = "0.3.0"
 readme = "README.md"
+description = "Classically postprocess noisy quantum samples to yield more accurate energy estimations"
```

Comment:
This is the reason for the `None` in the PyPI search results preview:
![screenshot_1726036823](https://github.com/user-attachments/assets/a5199682-a493-4641-b85c-fcc38b99a8f9)